lms/static/scripts/frontend_apps/components/ErrorDisplay.js

```diff
@@ -9,36 +14,6 @@ function toSentence(str) {
   return str.match(/[.!?]$/) ? str : str + '.';
 }
 
-/**
- * An `Error` or `Error`-like object. This allows this component to be used
- * by just passing through an `Error` without meddling with it, or manual
- * control of `message` and/or `details` if so desired.
- *
- * @typedef ErrorLike
- * @prop {string} [message]
- * @prop {object|string} [details] - Optional JSON-serializable details of the error
- * @prop {string} [errorCode] - Provided by back-end to identify error state
- * @prop {string} [serverMessage] - Explanatory message provided by backend that
- *   will be preferred over `message` if it is present.
- */
-
-/**
- * JSON-stringify `error.details` if it is extant and an object
- *
- * @param {ErrorLike} error
- */
-function formatErrorDetails(error) {
-  let details = error.details ?? '';
-  if (error?.details && typeof error.details === 'object') {
-    try {
-      details = JSON.stringify(error.details, null, 2 /* indent */);
-    } catch (e) {
-      // ignore
-    }
-  }
-  return details;
-}
-
 /**
  * @typedef ErrorDetailsProps
  * @prop {ErrorLike} error
```

Comment:
Unrelated to the changes in this PR, but the `ErrorDetailsProps` typedef should be a separate JSDoc block from the `ErrorDetails` component docs.

Reply:
Fixing since I'm fixing up comments and typedefs in this module now anyway.